doc/ref/debug.xml

```diff
@@ -158,6 +160,35 @@ returns the info level of <A>infoclass</A>.
 </Description>
 </ManSection>
 <P/>
+<ManSection>
+<Func Name="ShowUsedInfoClasses" Arg='infoclass'/>
+
+<Description>
+Called with argument <K>true</K>, makes GAP print the InfoClass and level of
```

Comment:
```suggestion
Called with argument <K>true</K>, makes &GAP; print the info class and level of
```

Reply:
@fingolfin I've edited this suggestion and the next ones to make the sentences more grammatical, my suggestions are independent of yours except for being on the same lines, which is why I have combined them.